### PR TITLE
Move to scratch

### PR DESCRIPTION
**Pull request by gianlucam76**

### CODE REVIEW

File: build/Dockerfile.amd64 [hunk 1]
```diff
@@ -22,9 +29,8 @@ LABEL name="Calico Operator" \
       description="Calico Operator manages the lifecycle of a Calico or Tigera Secure installation on Kubernetes or OpenShift" \
       maintainer="Laurence Man <laurence@tigera.io>"
 
-# Add in top-level license file
-RUN mkdir /licenses
-COPY LICENSE /licenses
+COPY --from=ubi /etc/passwd /etc/passwd
```
Comment (doublek) [inline build/Dockerfile.amd64:32]: Do you know who uses this file? I thought it was used only as a part of entrypoint script and perhaps now it's not required?

Comment (gianlucam76) [reply build/Dockerfile.amd64:32]: It does not appear to be needed. I can remove if that's what we want.

Comment (gianlucam76) [reply build/Dockerfile.amd64:32]: Not needed. I can remove it if that's what we want.

Comment (doublek) [reply build/Dockerfile.amd64:32]: If it's not strictly required I am okay removing it. I would like @tmjd's opinion as well.

Comment (gianlucam76) [reply build/Dockerfile.amd64:32]: Just FYI, removing following I was able to start from scratch and all worked (tested with crc).
 
-COPY --from=ubi /etc/passwd /etc/passwd
-COPY --from=ubi /licenses /licenses

Comment (doublek) [reply build/Dockerfile.amd64:32]: I think `-COPY --from=ubi /licenses /licenses` might be required but probably not the copy for `/etc/passwd`.

Comment (tmjd) [reply build/Dockerfile.amd64:32]: I don't know of a reason it is needed. If it works without I'm happy for it to be removed.
Awesome @gianlucam76, thanks for doing this.

File: build/Dockerfile.amd64 [hunk 2]
```diff
@@ -22,20 +23,13 @@ LABEL name="Calico Operator" \
       description="Calico Operator manages the lifecycle of a Calico or Tigera Secure installation on Kubernetes or OpenShift" \
       maintainer="Laurence Man <laurence@tigera.io>"
 
-# Add in top-level license file
-RUN mkdir /licenses
-COPY LICENSE /licenses
```
Comment (doublek) [inline build/Dockerfile.amd64:27]: I think license is still required for certification requirements. Sorry if I wasn't clear in my previous comment. My comment was to only remove /etc/passwd which shouldn't be required anymore.

File: build/Dockerfile.amd64 [hunk 3]
```diff
@@ -22,20 +33,13 @@ LABEL name="Calico Operator" \
...
     USER_NAME=operator
 
 # install operator binary
 COPY build/_output/bin/operator-amd64 ${OPERATOR}
 
-COPY build/bin /usr/local/bin
-RUN  /usr/local/bin/user_setup
-
-ENTRYPOINT ["/usr/local/bin/entrypoint"]
+ENTRYPOINT ["/usr/local/bin/operator"]
```
Comment (tmjd) [inline build/Dockerfile.amd64:43]: minor thing: Can we use `${OPERATOR}` here too?

**Pull request merged**